sass-spec.ts

```diff
@@ -11,36 +11,50 @@ const naughtyDirs = [
 ]
 
 async function runAllTests() {
-  const interactor = new Interactor(process.stdin, process.stdout)
-  const start = Date.now()
-  const args = await parseArgs(process.argv.slice(2))
-  const rootPath = args.root
-  const rootDir = await fromPath(rootPath)
-  const tabulator = new Tabulator(process.stdout, args.verbose)
+  let args_: CliArgs | undefined
```

Comment:
Might be unneeded? Typescript usually infers these correctly.

Reply:
In this case it didn't seem to (I think because it's referred to from within a callback?)